net/snort3/Makefile

```diff
@@ -39,6 +44,18 @@ define Package/snort3/description
   attacks.
 endef
 
+# Hyperscan and gperftools only builds for x86
```

Comment:
oops, i overlooked this. i think it's duplicate with the options below.
and for the dependencies update, it's definitely more than what you did in original commit.